main.py

```diff
@@ -135,8 +137,8 @@
             {'label': 'Hyperspeed Accelerators', 'value': 'engi_gloves'},
         ],
         value=[
-            'shred_idol', 'rip_idol', 'rip_glyph', 'shred_glyph', 'roar_glyph',
-            't7_2p', 'meta', 'berserking', 'engi_gloves'
+            'shred_idol', 'rip_glyph', 'shred_glyph', 'roar_glyph',
+            't7_2p', 'meta', 'berserking', 'engi_gloves', 't8_2p'
```

Comment:
Remove t8_2p as default, and add back in rip_idol, since having both Idols selected will trigger the Idol swapping implementation I put in.